xbmc/video/windows/GUIWindowVideoNav.h

```diff
@@ -49,7 +49,7 @@ class CGUIWindowVideoNav : public CGUIWindowVideoBase
   virtual bool OnAction(const CAction &action) override;
   virtual bool OnMessage(CGUIMessage& message) override;
 
-  virtual void OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr &info) override;
+  virtual void OnItemInfo(CFileItem* fileItem, ADDON::ScraperPtr &info) override;
```

Comment:
whatever it is going to be, it should not be hidden in a 1.5k loc change that suggests Android.